collector/meminfo_numa_linux.go

```diff
@@ -90,17 +97,31 @@ func getMemInfoNuma() (map[meminfoKey]float64, error) {
 		if err != nil {
 			return nil, err
 		}
-		for k, v := range numaInfo {
-			info[k] = v
+		metrics = append(metrics, numaInfo...)
+
+		file, err = os.Open(path.Join(node, "numastat"))
```

Comment:
Use a different variable for this file. Otherwise the `defer file.Close()` above won't close the `meminfo` but `numastat` and leak file descriptors.